submissions/SergSenras/js-dom/script.js

```diff
@@ -0,0 +1,108 @@
+}
+
+function clear(){
+  const pageContent = document.getElementsByTagName('main')[0];
+  while(pageContent.hasChildNodes()){
+    pageContent.removeChild(pageContent.firstChild);
+  }
+}
+
+function displayContent(key){
+  clear();
+  const container = document.getElementsByTagName('main')[0];
+  const fragment = document.createDocumentFragment();
+  var contentTowns = function(){
+    for(var i in towns){
+      const wrapper = document.createElement('div');
+      wrapper.className = 'item__box';
+      const headerEl = document.createElement('h2');
+      const textEl = document.createElement('p');
+      const imageEl = document.createElement('img');
+      imageEl.className = 'img__town img__common';
+      headerEl.innerText = towns[i].name;
+      textEl.innerText = towns[i].content;
+      imageEl.src = towns[i].image;
+      wrapper.appendChild(headerEl);
+      wrapper.appendChild(imageEl);
+      wrapper.appendChild(textEl);
+      fragment.appendChild(wrapper);
+    }
+    container.appendChild(fragment);
+  }
+  var contentPlot = function(){
+    clear();
+    const wrapper = document.createElement('div');
+    wrapper.className = 'item__box';
+    const textEl = document.createElement('p');
+    textEl.innerText = plot.content;
+    wrapper.appendChild(textEl);
+
+    container.appendChild(wrapper);
+  }
+  var contentMain = function(){
+    clear();
+    const wrapper = document.createElement('div');
+    wrapper.className = 'item__box';
+    const textEl = document.createElement('p');
+    const imageEl = document.createElement('img');
+    imageEl.className = 'img__common';
+    textEl.innerText = main.content;
+    imageEl.src = main.image;
+    wrapper.appendChild(imageEl);
+    wrapper.appendChild(textEl);
+    container.appendChild(wrapper);
+  }
+
+  switch(key){
+    case 'nav__main':
+      contentMain();
+      break;
+    case 'nav__plot':
+      contentPlot();
+      break;
+    case 'nav__towns':
+      contentTowns();
+      break;
+  }
+}
+
+document.addEventListener('DOMContentLoaded', function(){
+  var menu = document.getElementById('menu');
+  var nav = document.getElementById('nav');
+  var selectedOption = main;
+
+  // display content
+  displayContent('nav__main');
+
+  menu.addEventListener('click', function(e){
+    nav.classList.toggle('open');
+    e.stopPropagation();
+  });
+
+  var classname = document.getElementsByClassName('nav__item');
```

Comment:
`.getElementsByClassName` returns element, not class names. Please, refactor this variable.

Reply:
```suggestion
  var menuList = document.querySelector('.nav__list');
```
a little bit changed that